proxy/src/main/java/com/wavefront/agent/listeners/tracing/JaegerThriftCollectorHandler.java

```diff
@@ -133,6 +137,9 @@ public JaegerThriftCollectorHandler(String handle,
     this.preprocessor = preprocessor;
     this.sampler = sampler;
     this.alwaysSampleErrors = alwaysSampleErrors;
+    if (applicationName != null && !StringUtils.isBlank(applicationName.trim())) {
```

Comment:
!StringUtils.isBlank(applicationName) is sufficient (covers null, "", and any number of whitespaces)